core/src/main/java/feign/FeignException.java

```diff
@@ -271,149 +203,130 @@ static FeignException errorExecuting(Request request, IOException cause) {
   }
 
   public static class FeignClientException extends FeignException {
-    public FeignClientException(int status, String message, Request request, byte[] body,
```

Comment:
anything public, needs to be kept, add `@deprected` to them, but keep them in place

Reply:
done in the commit 2523ef1